lib/KCard/index.vue

```diff
@@ -407,7 +411,7 @@
   }
 
   .thumbnail-placeholder {
-    position: absolute;
+    position: relative;
```

Comment:
Why do we need this as relative now? :eyes: Not saying that its wrong, but would like to know the reason

Reply:
It needs to stay `absolute`, otherwise placeholder element won't be positoned as intended via other styles right below